docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
+
+import { registerRenderer } from 'handsontable/renderers/registry';
+import { autocompleteRenderer } from 'handsontable/renderers/autocompleteRenderer';
+import { baseRenderer } from 'handsontable/renderers/baseRenderer';
+import { checkboxRenderer } from 'handsontable/renderers/checkboxRenderer';
+import { htmlRenderer } from 'handsontable/renderers/htmlRenderer';
+import { numericRenderer } from 'handsontable/renderers/numericRenderer';
+import { passwordRenderer } from 'handsontable/renderers/passwordRenderer';
+import { textRenderer } from 'handsontable/renderers/textRenderer';
+
+import { registerValidator } from 'handsontable/validators/registry';
+import { autocompleteValidator } from 'handsontable/validators/autocompleteValidator';
+import { dateValidator } from 'handsontable/validators/dateValidator';
+import { numericValidator } from 'handsontable/validators/numericValidator';
+import { timeValidator } from 'handsontable/validators/timeValidator';
+
+import { registerLanguageDictionary } from 'handsontable/i18n/registry';
+```
+:::
+
+## Using modules
+
+To get the most out of using Handsontable modules:
+1. [Import the required modules](#importing-required-modules).
+2. Use optional modules of your choice:
+   - [Editor modules](#using-editor-modules)
+   - [Renderer modules](#using-renderer-modules)
+   - [Validator modules](#using-validator-modules)
+   - [Cell type modules](#using-cell-type-modules)
+   - [Plugin modules](#using-plugin-modules)
+   - [Translation modules](#using-translation-modules)
+3. Remove redundant code, using [tree shaking](#tree-shaking).
+
+### Importing required modules
+
+No matter which of the [optional modules](#list-of-all-modules) you use, the following modules are always required:
+
+- Handsontable's core functionalities
+- The default [cell type](@/guides/cell-types/cell-type.md): `text`
+- [moment.js](https://momentjs.com/)
+- [DOMpurify](https://www.npmjs.com/package/dompurify)
+- [core-js](https://www.npmjs.com/package/core-js)
+- Handsontable's CSS
+
+To import the required modules, follow the steps below.
+
+#### Step 1: Remove the full Handsontable import
+
+When using modules, you no longer need to import the full version of Handsontable.
+
+Remove the `handsontable` import:
+
+```js
+// remove this line
+import Handsontable from 'handsontable';
+```
+
+#### Step 2: Import the `handsontable/base` module
+
+Instead of the full Handsontable version, import the `handsontable/base` module.
 
-To use a specific editor you need to import it alongside the registering method. For instance, let's try adding the password editor.
+```js
+// import the `handsontable/base` module
+import Handsontable from 'handsontable/base';
+```
+
+#### Step 3: Import the CSS
+
+Handsontable's CSS file is not modularized, so you need to import it:
+
+```js
+import Handsontable from 'handsontable/base';
+
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.css';
```

Comment:
@kirszenbaum the above line is not standard JavaScript. In standard JavaScript, you would not be able to import a CSS file. The above line only works thanks to some bundlers (e.g. Webpack), but we don't know here what bundler is used by a developer.

However, the above remark should be solved on the "Installation" page, because the same mistake can be seen there: https://handsontable.com/docs/installation/#download-and-install-the-library

Because CSS is not modularized, I think that it does not even deserve a separate step here. The contents of this step should be merged with the previous step.

Reply:
Ok: ef4409c c56f46a

Reply:
Thanks, as I suggested, I think that the following comment should be also added to the "Installation" page:

```
// if you bundler allows it, import Handsontable's CSS
```